RELEASES.md

```diff
@@ -1164,7 +1164,7 @@ Compatibility Notes
 - [Changes in method matching against higher-ranked types][43880] This may cause
   breakage in subtyping corner cases. [A more in-depth explanation is available.][info/43880]
 - [rustc's JSON error output's byte position start at top of file.][42973]
-  Was previously relative to the rustc's internal `CodeMap` struct which
+  Was previously relative to the rustc's internal `SourceMap` struct which
```

Comment:
I don't think this sould be changed, because `CodeMap` was the name back then.

Reply:
You're right, it does speak in past tense.